processes/release_processes/README.MD

```diff
@@ -86,12 +86,8 @@ The workflow outputs its status to the console.
 
 * Check that the workflow succeeded.
 * Check that there's now a new Cromwell release listed [here](https://github.com/broadinstitute/cromwell/releases).
-* Look [in Travis](https://app.travis-ci.com/github/broadinstitute/cromwell/branches) for the release tag build that will publish Docker images for the new version.
-* Let `#dsp-batch-private` know that it's okay to resume merges to `develop`.
+* Let `#dsp-workflows-private` know that it's okay to resume merges to `develop`.
 * Announce release in `#dsp-batch`, set expectations about when the new version will be available in Terra.
-* It will take about an additional hour for the Docker image to build in Travis before its tag appears on the [Cromwell Docker Hub page](https://hub.docker.com/r/broadinstitute/cromwell/tags).
```

Comment:
Huh... were we actually still depending on Travis for this? Do we need to update our community release process to build and tag the images elsewhere?

Reply:
(We decided that we do need to look into this, but it's not a blocker for merging this PR.)

Reply:
See https://broadworkbench.atlassian.net/browse/WX-996